### DISCUSSION TITLE
Support Latent Consistency Models, generate image in 4 steps

### DISCUSSION THREAD

**Discussion by frankyifei** [comfyanonymous/ComfyUI#1817] [Ideas]
https://latent-consistency-models.github.io/
Seems quite promising and interesting. the quality of image seems decent in 4 steps.
![Teaser_v4](https://github.com/comfyanonymous/ComfyUI/assets/8104251/23464453-c7e0-4f9f-987e-100f3b4e6752)

**Comment by marclove** [comment]
This is possible already by loading the LCM as a Lora. I learned how to do it [here](https://www.youtube.com/watch?v=pw8eNpmhT5o).

It is a little awkward for the workflow. I haven't ventured into writing custom nodes yet, but it would be nice to have an LCM-specific node that let you toggle it on/off, since LCMs are great for exploring and prototyping parameters & workflows before doing a higher quality non-LCM generation.
